cli/azd/cmd/down.go

```diff
@@ -114,21 +114,10 @@ func (a *downAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 	startTime := time.Now()
 
 	destroyOptions := provisioning.NewDestroyOptions(a.flags.forceDelete, a.flags.purgeDelete)
-	destroyResult, err := infraManager.Destroy(ctx, destroyOptions)
-	if err != nil {
+	if _, err = infraManager.Destroy(ctx, destroyOptions); err != nil {
 		return nil, fmt.Errorf("deleting infrastructure: %w", err)
 	}
 
-	// Remove any outputs from the template from the environment since destroying the infrastructure
-	// invalidated them all.
-	for outputName := range destroyResult.Outputs {
-		a.env.DotenvDelete(outputName)
-	}
-
-	if err := a.env.Save(); err != nil {
-		return nil, fmt.Errorf("saving environment: %w", err)
-	}
-
```

Comment:
Don't we still need these to delete the values?

Reply:
> Don't we still need these to delete the values?

Added a comment after I opened the PR to explain what was going on here, but likely raced with your feedback.  We have this code in two places now, in both the provisioning manager and in down.go itself.  This is just removing the copy in down, since the work has already been doing in the provisioning manager's destroy function.